app/src/main/java/org/wikipedia/page/linkpreview/LinkPreviewDialog.kt

```diff
@@ -106,14 +137,12 @@ class LinkPreviewDialog : ExtendedBottomSheetDialogFragment(), LinkPreviewErrorV
         _binding = DialogLinkPreviewBinding.inflate(inflater, container, false)
         binding.linkPreviewToolbar.setOnClickListener { goToLinkedPage(false) }
         binding.linkPreviewOverflowButton.setOnClickListener {
-            PopupMenu(requireActivity(), binding.linkPreviewOverflowButton).run {
-                inflate(R.menu.menu_link_preview)
-                setOnMenuItemClickListener(menuListener)
-                show()
-            }
+            setupOverflowMenu()
         }
         L10nUtil.setConditionalLayoutDirection(binding.root, viewModel.pageTitle.wikiSite.languageCode)
 
+        binding.linkPreviewTopHandle.isVisible = viewModel.fromPlaces
```

Comment:
Don't we want to show the handle in all cases, not just Places?

Reply:
I'll ask Robin about this.